src/me/ryanhamshire/PopulationDensity/EntityEventHandler.java

```diff
@@ -18,11 +18,7 @@
 
 package me.ryanhamshire.PopulationDensity;
 
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Random;
+import java.util.*;
```

Comment:
Don't use wildcard imports, fix your IDE to not do this.

Reply:
what's the problem with that lol

Reply:
Easier to identify what the class is using, helpful when I'm debugging at times, especially when looking at repos I'm unfamiliar with (i.e. helpful for future maintainers/contributors). https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

Reply:
but intellij got you covered with that:
```
org.bukkit.block.data.type.Sign wall = (org.bukkit.block.data.type.Sign) block.getBlockData();
wall.setRotation(blockFace);
block.setBlockData(wall);

Sign s = (Sign)block.getState();
```
Same variable Sign, but assigned differently

Reply:
I'm not talking about specific usages/conflicts in the code - I just mean when I look at a class file, seeing what it imports tells me at a glance what it does/works with.